src/config.py

```diff
@@ -19,7 +19,7 @@
 AFTER_EASTER = pd.Timestamp("2021-04-06")
 SUMMER_SCENARIO_START = "2021-06-07"
 
-FAST_FLAG = "debug"
+FAST_FLAG = "full"
```

Comment:
```suggestion
FAST_FLAG = "debug"
```
We don't want to crash local machines by accident ;-)